tests/rtl/hardware_test.py

```diff
@@ -385,6 +388,73 @@ def test_hardware_axpy_double_pump_vec4():
     return test_hardware_axpy_double_pump(veclen=4)
 
 
+@rtl_test()
+def test_hardware_vadd_temporal_vectorization():
+    # TODO !!!!! THIS TEST STALLS WITH VITIS 2021.2 !!!!!
+    # But it works fine for 2020.2
```

Comment:
Could you add small description of the test?

Reply:
I've added a description to this test, and the others in `hardware_test.py`.